validator-tests/errormessage-hidden-removed-axe.json

```diff
@@ -0,0 +1,100 @@
+[
+  {
+    "description": "Ensures all ARIA attributes have valid values",
+    "help": "ARIA attributes must conform to valid values",
+    "helpUrl": "https://dequeuniversity.com/rules/axe/4.3/aria-valid-attr-value?application=webdriverjs",
```

Comment:
Question, non-blocker: Is the query param needed here? It seems like the URL is valid without it. Just curious.

Reply:
This is the file retrieved from aXe. I don't think we should be changing it.